DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/src/qrd_demo.cpp

```diff
@@ -189,10 +189,6 @@ int main(int argc, char *argv[]) {
       constexpr float kErrorThreshold = 1e-4;
       for (size_t row = 0; row < ROWS_COMPONENT; row++) {
         for (size_t col = 0; col < COLS_COMPONENT; col++) {
-          if (std::isnan(v_matrix[row][col][0]) ||
-              std::isnan(v_matrix[row][col][1])) {
-            count++;
-          }
```

Comment:
Can you explain why it is OK to delete this block of code?  I know std::isnan() produces a warning, but you can eliminate the warning by replacing it with sycl::isnan().

Reply:
The check is followed by a check of the squared difference against the reference data. If the output is nan and the reference data is not, this will cause the fail anyway.

Reply:
reverted and used sycl::isnan()